setup.py

```diff
@@ -1,20 +1,10 @@
-
-#from distutils.core import setup
+# -*- coding: utf-8 -*-
 from setuptools import setup
-import sys
-
-REQUIRES = ['oauth2']
-if sys.version_info < (2 , 6):
-    REQUIRES.append('simplejson')
 
-setup(name='cartodb',
-      author = 'Javi Santana',
-      author_email = 'jsantana@vizzuality.com',
-      description = 'client to access cartodb api',
-      version='0.8.2',
-      url='https://github.com/Vizzuality/cartodb',
-      install_requires=REQUIRES,
-      packages=['cartodb'],
-      requires = REQUIRES,
```

Comment:
This line is installation breaking. Currently you don't specify any requirements for your package, you just have a requirements.txt.

When installing things from pip, this line needs to be in place. The appropriate thing to do would be to read from `requirements.txt`.

To test, please have some examples, and create a project with them, using this as a dependency.

you can put this line in the requirements.txt of that examples project to try to reproduce. Also, remember that it should be a clean virtualenv.

`-e git+https://github.com/CartoDB/carto-python.git@3a4c514fcd71c94ddbcede710ecafb3db8d3abcb#egg=carto`